packages/arg-parser/src/arg-mapper.spec.ts

```diff
@@ -22,29 +22,29 @@ function optionsTest(cliOptions: CliOptions): { cs?: string, driver?: Partial<De
   return { cs: result.connectionString, driver: result.driverOptions };
 }
 
-describe('arg-mapper.mapCliToDriver', () => {
-  context('when cli args have authenticationDatabase', () => {
+describe('arg-mapper.mapCliToDriver', function() {
```

Comment:
I am actually impressed that not one test seemed to break due to some subtlety in this binding.